Iceberg-Plugin-GitHub/IceTipGitHubRepositoryPanel.class.st

```diff
@@ -23,11 +23,17 @@ IceTipGitHubRepositoryPanel >> configureBuilder: aBuilder [
 { #category : #'querying - github' }
 IceTipGitHubRepositoryPanel >> getGitHubRepository [
 
-	^ [ IceGitHubAPI new
-		beAnonymous;
-		getRepository: self userName project: self projectName ]
-			on: LGitNoCredentialsProvided
-			do: [ nil ]
+	^ [ [ IceGitHubAPI new
+		  beAnonymous;
+		  getRepository: self userName project: self projectName ]
+		  on: LGitNoCredentialsProvided
+		  do: [ nil ] ]
+		  on: IceGitHubError
+		  do: [ :error | 
+			"If not found, probably means a private repository => ignore it"
+			error messageText = 'Not Found'
```

Comment:
Can we have a better message text than "Not found". The user might wonder what is not found and how can proceed

Reply:
This is the message we have from the github request, so I don't think I have access there...
Anyways, I know the solution is sub optimal, so if you have any ideas I'm a taker ! :)

Reply:
My fault - so small on the phone that I though it is an assignment, not comparision. Yes you can then hardly change it - thanks for the feedback!